server.js

```diff
@@ -0,0 +1,80 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const nodemailer = require("nodemailer");
+const cors = require("cors");
+const path = require("path");
+const cluster = require("cluster");
+const numCPUs = require("os").cpus().length;
+
+require("dotenv").config();
+
+const PORT = process.env.port || 8080;
```

Comment:
It has taken me an hour of banging my head against my computer, but I've realized that `process.env.port` actually needs to be `process.env.PORT`. Remember that to save yourself the headache. :)